views/attendanceStatistics.njk

```diff
@@ -3,8 +3,15 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 {% from "components/datePicker/datePicker.njk" import datePicker %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 
- {% block beforeContent %}
+{% if shouldClearFormValues  %}
```

Comment:
What is `shouldClearFormValues`?

Reply:
Boolean, see the attendanceStatistic.js changes, you'll need to expand it to see it. Some are collapsed via the diff viewer.

Reply:
Doh